### PR TITLE
Direct swarm connect with topology

### PR DESCRIPTION
**Pull request by v-stickykeys**
This is helpful for testing/debugging swarm connections

### CODE REVIEW

File: packages/ipfs-topology/src/ipfs-topology.ts [hunk 1]
```diff
@@ -74,6 +74,15 @@ export class IpfsTopology {
     readonly logger: DiagnosticsLogger,
     readonly period: number = DEFAULT_PEER_DISCOVERY_PERIOD
   ) {}
+  
+  /**
+   * Connect to custom peer list
+   * @param peerList List of multiaddresses to swarm connect to
+   */
+  async connect(peerList: Array<string>) {
```
Comment (stbrody) [inline packages/ipfs-topology/src/ipfs-topology.ts:82]: is this function meant to be used somewhere?

Comment (ukstv) [reply packages/ipfs-topology/src/ipfs-topology.ts:82]: +1 to this.

**Review by ukstv — CHANGES_REQUESTED** [review]
Unclear how it fits.

File: packages/ipfs-topology/src/ipfs-topology.ts [hunk 2]
```diff
@@ -74,6 +74,15 @@ export class IpfsTopology {
...
+    const addressList = peerList.map((peer) => { return new Multiaddr(peer) });
```
Comment (ukstv) [inline packages/ipfs-topology/src/ipfs-topology.ts:83]:
```suggestion
    const addressList = peerList.map((peer) => new Multiaddr(peer));
```

**Review by ukstv — APPROVED** [review]

**Pull request merged**